content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,134 @@
+# data status
+
+Show changes in the data tracked by DVC in the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
+
+The `data status` command displays the state of the working directory and the
+changes with respect to the last Git commit (`HEAD`).
+
+It shows you what new changes have been committed to DVC, which haven't been
+committed, which files aren't being tracked by DVC and Git, and what files are
+missing from the cache. It shows you what's been going on with `dvc add`,
+`dvc commit` and `dvc repro` commands.
+
+The `dvc data status` command only outputs information, it won't modify or
+change anything in your working directory. It's a good practice to check the
+state of your repository before doing `git commit` so that you don't
```

Comment:
change to: `git commit` or `dvc commit`?

Reply:
`git commit` because there is no issue even with `dvc commit` unless you need to share it with others, you can re-commit it with dvc.

Reply:
hmm, it's the same with Git - you can recommit

Reply:
No, you’d have to amend the commit or add another one on top (at which point it’s not a recommit, the history is there). In DVC, even if you re-commit, dvc won’t know older versions of commit if it was not preserved via `git commit`.

Reply:
So you could say that dvc or `dvc commit` is forgiving in these accidental cases.

Reply:
that's fine @skshetry , not a strong opinion here ... it feels to me that status can be useful even before `dvc commit` (e.g. at least not to waste time, and / or make sure / see what is actually are getting to be saved, etc). Your call here.

Reply:
My intention was to say that it could be useful before sharing it via commit+push, but yeah, I do think we should generalize it.
It’s useful other times too. :)